webpack.config.js

```diff
@@ -1,26 +1,59 @@
 const webpack = require("webpack");
 const CopyWebpackPlugin = require("copy-webpack-plugin");
-const HtmlWebpackPlugin = require("html-webpack-plugin");
-const HtmlWebpackIncludeAssetsPlugin = require("html-webpack-include-assets-plugin");
+//const HtmlWebpackPlugin = require("html-webpack-plugin");
```

Comment:
We might be able to just remove these two plugins rather than comment them out.

Reply:
Yes, we might as well for now.